src/python/aqueduct_executor/operators/airflow/dag.template

```diff
@@ -11,7 +11,6 @@ VENV_REQUIREMENTS=[
     "cloudpickle",
     "distro",
     "google-cloud-bigquery",
-    "mysqlclient",
```

Comment:
See https://linear.app/aqueducthq/issue/ENG-2602/add-support-for-using-mysql-with-airflow for why this was causing issues. 

tl;dr we cant install mysqlclient by default